src/components/Results/DetectedIssueResources.tsx

```diff
@@ -36,63 +51,66 @@ const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
               </Grid>
             </AccordionSummary>
             <AccordionDetails>
-              <Grid container item xs direction="row">
+              <Grid container item xs direction="row" spacing={2}>
                 <Grid container item xs={6} direction="column">
                   <Grid item xs>
-                    <h4>dataRequirement:</h4>
+                    <h4>Requirements:</h4>
                   </Grid>
-                  <Grid item xs>
-                    type: {fhirpath.evaluate(response, 'dataRequirement.type')}
+                  <Grid item xs direction="row">
```

Comment:
If using `direction`, this needs to be a Grid `container`

Reply:
I realized that I actually didn't need a direction so I just made it a Grid item xs.